app/src/main/java/org/ea/sqrl/activites/SettingsActivity.java

```diff
@@ -60,6 +64,29 @@ protected void onCreate(Bundle savedInstanceState) {
         btnSettingsSave.setOnClickListener(v ->
             savePopupWindow.showAtLocation(savePopupWindow.getContentView(), Gravity.CENTER, 0, 0)
         );
+
+        txtSettingsIdleTimeout.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void afterTextChanged(Editable editable) {
+                if (editable.length() > 2) {
+                    Log.v("sengsational", "after " + editable.length());
```

Comment:
I don't know how we want to handle logging in general, but I don't see any verbose logging elsewhere in the app. The tag "sengsational" suggests that this was for debugging only. Should this be removed?

Reply:
Oops!  I meant to clean this up.  I put a unique tag on it so I can find it, hehe!

As for logging and comments, this code is devoid of any of those, for the most part.  When I code my own stuff, I put tons of logging and I leave it in the code base because when the code is built for `release`, it's not included, so doesn't bog down the devices of the "real" users.  A matter of taste, I suppose.  But I've always tried to bend my techniques to match the existing code, so if I remember, I pull out the logging.